src/Application.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Console\Application;
+use ExerciseCleaner\Command;
+
+$version = 'raw';
+
+$application = new Application('Exercise Cleaner', $version);
+
+$command = new Command();
+$application->add($command);
+//$application->setDefaultCommand($command->getName());//"cannot pass any argument or option to the default command" — https://symfony.com/doc/current/components/console/changing_default_command.html
```

Comment:
It's possible with a single command application

https://symfony.com/doc/5.0/components/console/single_command_tool.html

`$application->setDefaultCommand($command->getName(), true);`

Reply:
Done in d6a7b1f5f057287ac64a16a687da84dc9f247b5a...3afdf948d8d94142d3abf22838fa716adee9f9cb